src/Orc.CsvTextEditor/Controls/ViewModels/FindReplaceDialogViewModel.cs

```diff
@@ -0,0 +1,92 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="FindReplaceDialogViewModel.cs" company="WildGums">
+//   Copyright (c) 2008 - 2016 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.CsvTextEditor
+{
+    using System.Media;
+    using Catel.IoC;
+    using Catel.MVVM;
+    using Controls;
+    using Services;
+
+    public class FindReplaceDialogViewModel : ViewModelBase
```

Comment:
I don't think we need to make it too complicated
tools could be self-managed. We just need to be able to add them and remove.

Just try to make it simple. I think we just need to be able to add/remove tool (or dialog). And each tool should be self-managed